Search in admin on people page returns a 500

from the logs:

```
[2018-06-25 12:38:08,983] [ERROR] Internal Server Error: /cms/people/person/
Traceback (most recent call last):
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/Cellar/python/3.6.5/Frameworks/Python.framework/Versions/3.6/lib/python3.6/contextlib.py", line 52, in inner
    return func(*args, **kwds)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/views/decorators/cache.py", line 43, in _cache_controlled
    response = viewfunc(request, *args, **kw)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/wagtail/admin/urls/__init__.py", line 102, in wrapper
    return view_func(request, *args, **kwargs)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/wagtail/admin/decorators.py", line 34, in decorated_view
    return view_func(request, *args, **kwargs)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/wagtail/contrib/modeladmin/options.py", line 315, in index_view
    return view_class.as_view(**kwargs)(request)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/utils/decorators.py", line 67, in _wrapper
    return bound_func(*args, **kwargs)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/utils/decorators.py", line 63, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/wagtail/contrib/modeladmin/views.py", line 246, in dispatch
    self.queryset = self.get_queryset(request)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/wagtail/contrib/modeladmin/views.py", line 583, in get_queryset
    request, qs, self.query)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/wagtail/contrib/modeladmin/views.py", line 273, in get_search_results
    queryset = queryset.filter(reduce(operator.or_, or_queries))
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/db/models/query.py", line 784, in filter
    return self._filter_or_exclude(False, *args, **kwargs)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/db/models/query.py", line 802, in _filter_or_exclude
    clone.query.add_q(Q(*args, **kwargs))
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1250, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1270, in _add_q
    current_negated, allow_joins, split_subq)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1276, in _add_q
    allow_joins=allow_joins, split_subq=split_subq,
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1154, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg)
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1034, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "/Users/lucie/.local/share/virtualenvs/network-6leWfGjP/lib/python3.6/site-packages/django/db/models/sql/query.py", line 1352, in names_to_path
    "Choices are: %s" % (name, ", ".join(available)))
django.core.exceptions.FieldError: Cannot resolve keyword 'n' into field. Choices are: affiliations, bio, expires, featured, id, image, internet_health_issues, interview_url, leaders_related_homepage, linkedin_url, location, name, order, partnership_logo, publish_after, quote, role, twitter_url
Internal Server Error: /cms/people/person/
```

--- a/network-api/networkapi/people/admin.py
+++ b/network-api/networkapi/people/admin.py
@@ -44,7 +44,7 @@ class WagtailPersonAdmin(ModelAdmin):
     exclude_from_explorer = True  # or True to exclude pages of this type from Wagtail's explorer view
     list_display = ('name', 'role', 'location')
     list_filter = ('internet_health_issues',)
-    search_fields = ('name')
+    search_fields = ('name',)
 
 
 modeladmin_register(WagtailPersonAdmin)

